### PR TITLE
BREAKING CHANGE: remove pre-1.0.0 deprecations

### PR DESCRIPTION
**Pull request by rubencarvalho**
## Description

This PR has all combined deprecation removals.
## Related issue(s)

- [SWC-302](https://jira.corp.adobe.com/browse/SWC-302)

## Motivation and context

We want to kick-off 1.0.0 with a clean(er) slate 😄 

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [x] Breaking change (fix or feature that would cause existing functionality to change)
-   [ ] Chore (minor updates related to the tooling or maintenance of the repository, does not impact compiled assets)

## Checklist

-   [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
-   [x] My code follows the code style of this project.
-   [x] If my change required a change to the documentation, I have updated the documentation in this pull request.
-   [x] I have read the **[CONTRIBUTING](<(https://github.com/adobe/spectrum-web-components/blob/main/CONTRIBUTING.md)>)** document.
-   [x] I have added tests to cover my changes.
-   [x] All new and existing tests passed.
-   [x] I have reviewed at the Accessibility Practices for this feature, see: [Aria Practices](https://www.w3.org/TR/wai-aria-practices/)

## Best practices

This repository uses conventional commit syntax for each commit message; note that the GitHub UI does not use this by default so be cautious when accepting suggested changes. Avoid the "Update branch" button on the pull request and opt instead for rebasing your branch against `main`.

### CODE REVIEW

File: web-test-runner.utils.js [hunk 1]
```diff
@@ -157,14 +157,14 @@ const vrtHTML =
     </html>`;
 
 export let vrtGroups = [];
-const themeVariants = ['spectrum', 'express', 'spectrum-two'];
+const systemVariants = ['spectrum', 'express', 'spectrum-two'];
 const colors = ['lightest', 'light', 'dark', 'darkest'];
```
Comment (Rajdeepc) [inline web-test-runner.utils.js:161]: Should we remove `lightest` and `darkest` from here too?

File: web-test-runner.utils.js [hunk 2]
```diff
@@ -157,14 +157,14 @@ const vrtHTML =
...
 const scales = ['medium', 'large'];
 const directions = ['ltr', 'rtl'];
-themeVariants.forEach((themeVariant) => {
+systemVariants.forEach((systemVariant) => {
     colors.forEach((color) => {
         if (
-            themeVariant === 'spectrum-two' &&
+            systemVariant === 'spectrum-two' &&
             (color === 'lightest' || color === 'darkest')
```
Comment (Rajdeepc) [inline web-test-runner.utils.js:168]: Reference here too!

File: tools/theme/test/theme-lazy.test.ts [hunk 3]
```diff
@@ -125,11 +117,9 @@ describe('Themes - lazy', () => {
         }
     });
     it('loads w/ lazy themes', async () => {
-        const el = await fixture<Theme>(
-            html`
-                <sp-theme color="lightest" scale="large"></sp-theme>
-            `
-        );
+        const el = await fixture<Theme>(html`
+            <sp-theme color="lightest" scale="large"></sp-theme>
```
Comment (Rajdeepc) [inline tools/theme/test/theme-lazy.test.ts:121]: lets remove `lightest` and add `light`?

File: tools/theme/test/theme-lazy.test.ts [hunk 4]
```diff
@@ -103,11 +97,9 @@ describe('Themes - lazy', () => {
         }
     });
     it('loads w/ not enough themes', async () => {
-        const el = await fixture<Theme>(
-            html`
-                <sp-theme color="lightest" scale="large"></sp-theme>
-            `
-        );
+        const el = await fixture<Theme>(html`
+            <sp-theme color="lightest" scale="large"></sp-theme>
```
Comment (Rajdeepc) [inline tools/theme/test/theme-lazy.test.ts:101]: lets remove lightest and add light?

File: tools/theme/test/theme-lazy.test.ts [hunk 5]
```diff
@@ -86,11 +82,9 @@ describe('Themes - lazy', () => {
         expect(el.scale).to.equal('medium');
     });
     it('loads w/ no themes', async () => {
-        const el = await fixture<Theme>(
-            html`
-                <sp-theme color="lightest" scale="large"></sp-theme>
-            `
-        );
+        const el = await fixture<Theme>(html`
+            <sp-theme color="lightest" scale="large"></sp-theme>
```
Comment (Rajdeepc) [inline tools/theme/test/theme-lazy.test.ts:86]: lets remove lightest and add light?

File: tools/theme/src/Theme.ts [hunk 6]
```diff
@@ -522,11 +485,6 @@ function checkForIssues(
             }
         };
 
-        if (hasThemeAttribute) {
-            issues.push(
-                `DEPRECATION NOTICE: the "theme" attribute has been deprecated in favor of "system". For more information, see: https://opensource.adobe.com/spectrum-web-components/tools/theme/`
-            );
-        }
         if (['lightest', 'darkest'].includes(color || '')) {
```
Comment (Rajdeepc) [inline tools/theme/src/Theme.ts:488]: This can be remove?

File: scripts/spectrum-vars.js [hunk 7]
```diff
@@ -171,9 +171,13 @@ const foundVars = await findUsedVars();
 spectrumPaths.forEach((spectrumPath, i) => {
     const packageDir = ['styles'];
     const isExpress = i === 1;
-    if (isExpress) packageDir.push('express');
+    if (isExpress) {
+        packageDir.push('express');
+    }
     themes.forEach((theme) => {
-        if (isExpress && ['lightest', 'darkest'].includes(theme)) return;
+        if (isExpress && ['lightest', 'darkest'].includes(theme)) {
```
Comment (Rajdeepc) [inline scripts/spectrum-vars.js:178]: can we discard `lightest` and `darkest` from here too?

**Review by Rajdeepc — CHANGES_REQUESTED** [review]

File: tools/theme/src/Theme.ts (hunk 6 above)
Comment (rubencarvalho) [reply tools/theme/src/Theme.ts:488]: It is still supported but is deprecated, so I think we can keep the warning.

File: scripts/spectrum-vars.js (hunk 7 above)
Comment (rubencarvalho) [reply scripts/spectrum-vars.js:178]: I would prefer not to change this file, as it was recently reviewed during the S2 "plumbing" work, and we decided to maintain the same structure as Express.

File: web-test-runner.utils.js (hunk 2 above)
Comment (rubencarvalho) [reply web-test-runner.utils.js]: Keeping lightest and darkest in our test suites.

**Review by Rajdeepc — APPROVED** [review]

**Pull request merged**